awswrangler/athena/_utils.py

```diff
@@ -84,10 +84,14 @@ def _start_query_execution(
     workgroup: Optional[str] = None,
     encryption: Optional[str] = None,
     kms_key: Optional[str] = None,
+    client_request_token: Optional[str] = None,
     boto3_session: Optional[boto3.Session] = None,
 ) -> str:
     args: Dict[str, Any] = {"QueryString": sql}
 
+    # Athena client request token
+    args["ClientRequestToken"] = client_request_token if client_request_token else str(uuid.uuid4())
```

Comment:
If not client request token is provided, I think we should just let `boto3` take care of this, and just not have this parameter in the API call.

Reply:
+1